contrib/cirrus/lib.sh

```diff
@@ -181,6 +181,12 @@ setup_rootless() {
     # Maintain access-permission consistency with all other .ssh files.
     install -Z -m 700 -o $ROOTLESS_USER -g $ROOTLESS_USER \
         /root/.ssh/known_hosts /home/$ROOTLESS_USER/.ssh/known_hosts
+
+    if [[ -n "$ROOTLESS_USER" ]]; then
```

Comment:
Could you explain why this had to be moved? And if possible, could you revert the `showrun echo`? Those are helpful for debugging when all else fails.

Reply:
I think without this update, the rootless_user stuff would cause issues in running the tests for the farm set up. @cevich helped me with this part so he might be able to explain the need better.

Reply:
IIRC this was a copy-paste move from `setup_environment.sh`.  In hind-sight, the conditional can probably be dropped, as it may be confusing.  IIRC (please check) these steps are always required at the end of this function.  Otherwise the next time `lib.sh` is sourced, the caller will not have ROOTLESS_USER/ROOTLESS_UID set.

Reply:
@cevich so I should drop this change then?

Reply:
oh no sorry, I just mean the "if" can be dropped.  We still need those commands to run.